src/main/java/org/kuali/kfs/kns/web/struts/action/KualiDocumentActionBase.java

```diff
@@ -1109,18 +1131,17 @@ public ActionForward refresh(ActionMapping mapping, ActionForm form, HttpServlet
      * @param kualiForm
      */
     @SuppressWarnings("unchecked")
-    protected void refreshAdHocRoutingWorkgroupLookups(HttpServletRequest request, KualiDocumentFormBase kualiForm) {
-        for (Enumeration<String> i = request.getParameterNames(); i.hasMoreElements(); ) {
-            String parameterName = i.nextElement();
+    protected void refreshAdHocRoutingWorkgroupLookups(final HttpServletRequest request, final KualiDocumentFormBase kualiForm) {
+        for (final Enumeration<String> i = request.getParameterNames(); i.hasMoreElements(); ) {
+            final String parameterName = i.nextElement();
             if ("newAdHocRouteWorkgroup.recipientName".equals(parameterName)
                     && StringUtils.isNotEmpty(request.getParameter(parameterName))) {
                 //check for namespace
                 String namespace = KFSConstants.CoreModuleNamespaces.KFS;
-                // CU customization: use StringUtils.isNotBlank instead of StringUtils.isNotEmpty and trim which is not null safe
-                if (StringUtils.isNotBlank(request.getParameter("newAdHocRouteWorkgroup.recipientNamespaceCode"))) {
+                if (StringUtils.isNotEmpty(request.getParameter("newAdHocRouteWorkgroup.recipientNamespaceCode").trim())) {
```

Comment:
We have a null safety customization on this line that needs to be preserved.

Reply:
updated